src/common/solution-info.ts

```diff
@@ -33,4 +34,15 @@ function versionDetail(version: string): string {
   }
 
   throw new Error(`Illegal version string '${version}'.`);
-}
\ No newline at end of file
+}
+
+function versionShort(version: string): string {
```

Comment:
Let `versionDetail` to use this code too.

Reply:
done